tests/playbooks/tests_network_state.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+
+- name: Test configuring ethernet devices
+  hosts: all
+  vars:
+    type: veth
+    interface: ethtest0
+
+
+  tasks:
+    - name: "set type={{ type }} and interface={{ interface }}"
+      set_fact:
+        type: "{{ type }}"
+        interface: "{{ interface }}"
+    - include_tasks: tasks/show_interfaces.yml
+    - include_tasks: tasks/manage_test_interface.yml
+      vars:
+        state: present
+    - include_tasks: tasks/assert_device_present.yml
+
+    - name: Import network role
+      import_role:
+        name: linux-system-roles.network
+      vars:
+        network_state:
+          interfaces:
+            - name: "{{ interface }}"
+              type: ethernet
+              state: up
+              ipv4:
+                address:
+                  - ip: 192.0.2.251
+                    prefix-length: 30
+                dhcp: false
+          routes:
+            config:
+            - destination: 192.0.2.100/30
```

Comment:
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax
"Indent list contents beyond the list definition"